<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pq_page_export_csv_push_button</class>
 <widget class="QDialog" name="pq_page_export_csv_push_button">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>1034</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(95, 87, 87);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="6">
    <widget class="QComboBox" name="pq_page_select_topic_area_combo_box">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>SELECT TOPIC AREA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="pq_page_select_status_combo_box">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>SELECT STATUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Answered</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unanswered</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="pq_page_select_action_combo_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentText">
      <string>SELECT ACTION</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>SELECT ACTION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Information</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Review</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="pq_page_pq_ref_line_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter PQ Ref</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="pq_page_logout_push_button">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pq_page_from_date_edit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>yyyy/mm/dd</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="pq_page_apply_filters_push_button">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(225, 228, 239);</string>
     </property>
     <property name="text">
      <string>Apply Filters</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="pq_page_keywords_line_edit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter Keywords (e.g. heart,dog,chocolate)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="pq_page_apply_changes_push_button">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 240, 237);
color: rgb(255, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Apply Changes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pq_page_refresh_button">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(225, 228, 239);</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pq_page_home_push_button">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="9">
    <widget class="QTableWidget" name="pq_page_table">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(243, 243, 243);</string>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>210</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>PQ Ref</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Submitted</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Question</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Answer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Topic Area</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="pq_page_to_date_edit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>yyyy/mm/dd</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgb(241, 241, 241);</string>
     </property>
     <property name="text">
      <string>Date From</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(238, 238, 238);</string>
     </property>
     <property name="text">
      <string>Date To</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pq_page_export_to_csv_push_button">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Export CSV</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
